README.md

```
@@ -0,0 +1,129 @@
+2. Press **Cmd+Shift+K** (hotkey).
+3. KeyFlip automatically converts the clipboard text to the "to" layout (e.g., Ukrainian) and pastes it wherever your cursor is.
+
+---
+
+## 🔧 Notes
+
+* Only **macOS** is currently supported.
+* The default hotkey is **Cmd+Shift+K** and cannot be changed via CLI.
+* Config file is stored at:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)